### ISSUE TITLE
Debian packaging

### ISSUE THREAD

**Issue by hyiltiz** [alacritty/alacritty#3288]
Added the Pop!_OS PPA to `apt` in Debian testing, but the key had problems, preventing installation. Since Debian is one of the major distributions, packaging for Debian would also cover other downstream distros (such as Ubuntu etc.) Maybe the procedure is quite similar to what Pop!_OS PPA or the Fedora package is already doing? Is it planned to provide official packages for Debian? Would it make sense to add this to the Milestone?

This issue is similar but not exactly a duplicate of #2038.

```
root@Debian:~# add-apt-repository ppa:mmstick76/alacritty
 Repository for Ubuntu that provides standalone Alacritty packages built as part of the CI/CD infrastructure for Pop!_OS. Sources currently originate from https://github.com/pop-os/alacritty
 More info: https://launchpad.net/~mmstick76/+archive/ubuntu/alacritty
Press [ENTER] to continue or ctrl-c to cancel adding it

gpg: keybox '/tmp/tmphm93l14l/pubring.gpg' created
gpg: /tmp/tmphm93l14l/trustdb.gpg: trustdb created
gpg: key 8AC9B4BBBAB4900B: public key "Launchpad PPA for Michael Aaron Murphy" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: no valid OpenPGP data found.

root@Debian:~# apt update
Hit:1 http://ftp.us.debian.org/debian testing InRelease
...
Get:4 http://ppa.launchpad.net/mmstick76/alacritty/ubuntu focal InRelease [15.9 kB]
Err:4 http://ppa.launchpad.net/mmstick76/alacritty/ubuntu focal InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 8AC9B4BBBAB4900B
Reading package lists... Done
W: GPG error: http://ppa.launchpad.net/mmstick76/alacritty/ubuntu focal InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 8AC9B4BBBAB4900B
E: The repository 'http://ppa.launchpad.net/mmstick76/alacritty/ubuntu focal InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

**Comment by kchibisov** [comment]
We're developers, not package maintainers, etc. You're asking the wrong folks tbh and should better ask debian folks who maintain packages or open a ticket for them with a request to add Alacritty.

Ideally we should just provide sources for our releases and distros could use them for packaging. This thing we're doing already and you can see them on our release page.

The links you see in README.md about packages are contributed to us for informational reasons. Like the Pop!_OS PPA you've mentioned.

Also if you have problems with packages, it's up to package maintainer, and not to us.

**Comment by hyiltiz** [comment]
I understand that packaging and maintaining distro specific packages is by no means the dev's responsibilities. I was unaware that the list of packages in `README.md` was community maintained packages rather than officially supported packages. I found Debian in fact already has a ticket filed 3 years ago, but was blocked by #357.

**Comment by kchibisov** [comment]
>was community maintained packages rather than officially supported packages.

It's a list of official packages in distros, which is contributed from time to time to us.

**Comment by chrisduerr** [comment]
>  found Debian in fact already has a ticket filed 3 years ago, but was blocked by #357.

Please note that Debian packaging is not really blocked by #357. Even if this was resolved, it is unlikely that Debian will accept Alacritty as a package with their current packaging guidelines.

They already have problems with lining up their packaging goals with the existing rust ecosystem on Rust and Alacritty is pretty much as difficult as it gets for them. So this isn't likely going to happen with Debian and Fedora at least.

**Comment by kalexm** [comment]
WebGL in Chrome runs smooth 3D is working. Which Linux App would prove that OpenGL works?
